.github/ISSUE_TEMPLATE/user-facing-feature.md

```diff
@@ -1,40 +1,51 @@
 ---
 name: User facing feature
-about: For internal use
-title: ''
-labels: ''
+about: For internal use only; Open a user-facing feature tracking file
+title: "[FEATURE] - "
+labels: ["type: internal-feature", "user type: internal"]
 assignees: ''
 
 ---
 
-## What - The user facing feature being implemented
+## What
+<!-- What the end user will see -->
 
 ## Why
+<!-- The reason why this user-facing feature is being implemented -->
 
-## Personas - Who will this affect?
+## Personas
+<!-- Who will this affect? -->
 
-- [ ] SPO
-- [ ] Dapp Devs
+- [ ] SPOs
+- [ ] dApp Devs
 - [ ] Exchanges
-
-## Acceptance Criteria
+- [ ] Wallets
+- [ ] 3rd party tools
 
 ## Definition of done
 
-- [ ] Code review
+- [ ] Acceptance Criteria + User Stories & DoD created and singed-off (by PO, dev & test owners)
```

Comment:
I don't know what a "User Story" is. Can you add a concise definition please.

Reply:
I added a link to a doc explaining what AC, US & DoD are and how they are used.